Library/Homebrew/formula_text_auditor.rb

```diff
@@ -32,12 +32,12 @@ def to_s
     end
 
     def line_number(regex, skip = 0)
-      index = @lines.drop(skip).index { |line| line =~ regex }
+      index = @lines.drop(skip).index { |line| line.match?(regex) }
       index ? index + 1 : nil
     end
 
     def reverse_line_number(regex)
```

Comment:
Can we delete this method? I think the only use was removed in https://github.com/homebrew/brew/commit/413a7e5daebb93f75f8b4839dfd61e42d49b070e#diff-0718ff8226ebdfd9a6b515e3c2017968bf77c4dad21e7cd6bdbd84c7d6c479c4L292

Reply:
Looks like the whole class can be removed and its reference replaced with a file read as the only thing called is `to_s`